### ISSUE TITLE
Text alignment on y-axis label only shown correctly for ALIGN_NORMAL

### ISSUE THREAD

**Issue by ilahja89** [patrykandpatrick/vico#692]
### How to reproduce

Set the axis alignment in e.g. in one of the line chart examples to ALIGN_OPPOSITE
```
CartesianChartHost(
            chart =
                rememberCartesianChart(
                    rememberLineCartesianLayer(
                    ...
                    startAxis = rememberStartAxis(
                        label = rememberTextComponent(textAlignment = Layout.Alignment.ALIGN_OPPOSITE)
                    ),
                    bottomAxis = rememberBottomAxis(),
                ),
            model = model,
        )
```

### Observed behavior

The axis label disappears instead of being right aligned

### Expected behavior

Label is shown with the correct alignment. By now i didn't find a workaround to resolve this

### Vico version(s)

Latest unstable version

### Android version(s)

Android 14

**Comment by patrickmichalik** [comment]
Hello! Thanks for the report. We’ll look into it.

**Comment by patrickmichalik** [comment]
[Vico 2.0.0 Alpha 20](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.20) resolves this. Cheers!

**Comment by ilahja89** [comment]
@patrickmichalik Any news on that? could you reproduce it?

**Comment by Gowsky** [comment]
Hello @ilahja89, `textAlignment` applies only when there are multiple lines of text with different lengths. Please open a feature request. As long as you have a monospace font (like in your screenshot), you can use a workaround to add a leading space before positive values.

**Comment by Gowsky** [comment]
It’s fixed in [v1.15.0](https://github.com/patrykandpatrick/vico/releases/tag/v1.15.0).
